sdk/storage/storage-blob/package.json

```diff
@@ -71,6 +71,8 @@
     "build:browserzip": "gulp zip",
     "build:es6": "tsc -p tsconfig.json",
     "build:nodebrowser": "rollup -c 2>&1",
+    "build:js-samples": "npm run clean && npm run build:es6 && npm run build:nodebrowser",
+    "build:ts-samples": "npm run clean && cd samples && tsc -p . ",
```

Comment:
For debugging samples, "build:js-samples" & "build:ts-samples" are being used as prelaunch tasks in VSCode Debug config.
Is the nomenclature `build:js-samples` fine ? 


Or something like `build-samples:js` & `build-samples:ts`?
(to be in sync with the alphabetical order of the scripts in package.json and still keep both of them together)